src/axom/quest/examples/quest_step_file.cpp

```diff
@@ -0,0 +1,2218 @@
+// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/config.hpp"
+#include "axom/core.hpp"
+#include "axom/slic.hpp"
+#include "axom/primal.hpp"
+#include "axom/mint.hpp"
+
+#include "axom/CLI11.hpp"
+#include "axom/fmt.hpp"
+
+#include "opencascade/BRepAdaptor_Curve.hxx"
+#include "opencascade/BRepBuilderAPI_MakeFace.hxx"
+#include "opencascade/BRepBuilderAPI_NurbsConvert.hxx"
+#include "opencascade/BRepMesh_IncrementalMesh.hxx"
+#include "opencascade/BRepTools.hxx"
+#include "opencascade/BRep_Tool.hxx"
+#include "opencascade/Geom2d_BSplineCurve.hxx"
+#include "opencascade/Geom2d_Curve.hxx"
+#include "opencascade/Geom2dConvert.hxx"
+#include "opencascade/GeomAbs_CurveType.hxx"
+#include "opencascade/Geom_BSplineSurface.hxx"
+#include "opencascade/Geom_RectangularTrimmedSurface.hxx"
+#include "opencascade/Geom_Surface.hxx"
+#include "opencascade/Interface_Static.hxx"
+#include "opencascade/Poly_Triangulation.hxx"
+#include "opencascade/Precision.hxx"
+#include "opencascade/STEPControl_Reader.hxx"
+#include "opencascade/TColgp_Array1OfPnt.hxx"
+#include "opencascade/TColgp_Array2OfPnt.hxx"
+#include "opencascade/TopAbs.hxx"
+#include "opencascade/TopExp.hxx"
+#include "opencascade/TopExp_Explorer.hxx"
+#include "opencascade/TopLoc_Location.hxx"
+#include "opencascade/TopTools_IndexedMapOfShape.hxx"
+#include "opencascade/TopoDS.hxx"
+#include "opencascade/TopoDS_Edge.hxx"
+#include "opencascade/TopoDS_Face.hxx"
+#include "opencascade/TopoDS_Shape.hxx"
+#include "opencascade/TopoDS_Wire.hxx"
+#include <iostream>
+
+/**
+ * /file quest_step_file.cpp
+ * /brief Example that loads in a STEP file and converts the surface patches and curves to Axom's NURBS representations
+ *
+ * This example reads in STEP files representing trimmed NURBS meshes using OpenCASCADE, 
```

Comment:
Is it `Open CASCADE` or `OpenCASCADE` (no space)? I see both forms in your changes.

Reply:
Based on https://en.wikipedia.org/wiki/Open_Cascade_Technology, it looks like we should use "Open Cascade". I'll change it throughout.

Reply:
done